_config.yml

```diff
@@ -64,6 +64,10 @@ footer:
 
   # Icon between year and copyright info.
   icon: user
```

Comment:
2 spaces indents here, please:
```yml
footer:
...
  # Icon between year and copyright info.
  icon: heart
    # If you want to animate the Icon, set it to true.
    animated: false
    # Change the color of icon, using Hex Code.
    color: "#808080"
...
```

Reply:
I do it many times, and bad indent still.

Reply:
`footer.icon.animated` not work or what u mean?

Reply:
```
footer:
  # Specify the date when the site was setup.
  # If not defined, current year will be used.
  #since: 2015

  # Icon between year and copyright info.
  icon: heart
    # If you want to animate the icon, set it to true.
    animated: false
    # Change the color of icon, using Hex Code.
    color: "#808080"
```
YAMLException: bad indentation of a mapping entry at line 68, column 5:
        animated:  false

Reply:
Now that user can change the icon & it's animation, I think it's better to set `icon: user` as default.

Reply:
> YAMLException: bad indentation of a mapping entry at line 68, column 5:
animated: false

Oh, of course it will be error, because `icon: heart` already setted up. Need to something like this:

```yml
footer:
...
  # Icon between year and copyright info.
  icon:
    image: heart
    # If you want to animate the Icon, set it to true.
    animated: false
    # Change the color of icon, using Hex Code.
    color: "#808080"
...
```

Reply:
In this way, need to refactor `footer.icon` in `footer.swig`, are you sure?

Reply:
Yes, need to refactor and it then will breaking changes. So, version must be 6.1 in next NexT release.